app/apps/summons/models.py

```diff
@@ -118,7 +118,18 @@ class SummonedPerson(models.Model):
     )
 
     def __str__(self):
```

Comment:
Ga je deze string representatie nog gebruiken onder de status tekst in de interface?